controller/iteration.go

```diff
@@ -37,7 +38,7 @@ func NewIterationController(service *goa.Service, db application.DB, config Iter
 
 // CreateChild runs the create-child action.
 func (c *IterationController) CreateChild(ctx *app.CreateChildIterationContext) error {
-	_, err := login.ContextIdentity(ctx)
+	currentUser, err := login.ContextIdentity(ctx)
```

Comment:
if we start to make this kind of restrictions, it might be better to rely on pre-defined sql queries rather than multiple calls to the login model + space model + .... Not a blocker for this change